src/store/ducks/plans.ts

```diff
@@ -455,14 +455,16 @@ export function getPlansArray(
   intervention: InterventionType = InterventionType.FI,
   status: string[],
   reason: string | null = null,
-  jurisdictions: string[] = []
+  jurisdictions: string[] = [],
+  jurisdictionName: string | null = null
```

Comment:
This looks good to me! Eventually, I could see being expanded to be like a `jurisdictionProperty` object being passed in as a flexible key/value for this function to check the Jurisdictions against.

Reply:
I actually think it might be a good idea to pass all the arguments as a single object, since the list of arguments has grown to be too long for a function.

Reply:
> I actually think it might be a good idea to pass all the arguments as a single object, since the list of arguments has grown to be too long for a function.

PRs are welcome! :)

Reply:
Yeap, should i do that from here? might take longer to clear this of the pending list.

Reply:
I'm fine with you creating a new issue for that